cpp/data/sql_updates/vufind.1

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE vufind.user ADD ub_subscribed_to_newsletter BOOL NOT NULL DEFAULT FALSE;
```

Comment:
change prefix here as well

Reply:
Why is this conversation resolved? Nothing has changed.
Conversations should normally only be resolved by the person who begins them.